build/implementations/get-testcase-assertions.js

```diff
@@ -0,0 +1,17 @@
+const getAssertionSource = require('./get-assertion-source')
+
+/**
+ * Get all assertions for given testcase
+ *
+ * @param {Array<Object>} assertions assertions
+ * @param {String} relativeUrl relative url of testcase
+ */
+const getTestcaseAssertions = (assertions, relativeUrl) => {
```

Comment:
Instead of having this pass the relative URL, why don't you have the test case passed in, have this method figure out the relative URL and decide based on that. Seems like better encapsulation that way.